### DISCUSSION TITLE
to create Hubs cloud aws in Private subnet

### DISCUSSION THREAD

**Discussion by ks-aa** [mozilla/hubs#3992] [Hubs Cloud Questions] [answered]
I'm trying to use Hubs Cloud aws (enterprise) for business purposes.
Security rules require EC2 to be placed on a private subnet.
Is it possible to use it by modifying the CloudFormation template?
Or is it impossible due to the specifications of the application?

**Comment by rawnsley** [comment, marked as answer]
Do you mean Hubs security rules? The private subnet is still accessible ultimately through the public domain name you give it.

**Reply by ks-aa** [reply]
Thank you for your reply.
What I wanted to do was remove the public IP from EC2 and still have a way to get Hubs to work.

Currently, when created with CloudFormation, public IP is given to EC2 for APP and Stream, but at the same time there is a possibility of being attacked from the outside.
My company's security rules require EC2 to be placed on a private subnet to prevent external attacks.
So I asked if there was a way Hubs would work properly while placing EC2 on a private subnet.

**Reply by rawnsley** [reply]
Ah OK I understand better now. This is outside of my Hubs Cloud knowledge sorry.

**Reply by ks-aa** [reply]
understood.
I will try various things.
Thank you very much.

**Reply by johnshaughnessy** [reply]
I don't know the answer to this either. @ks-aa were you able to solve your own problem?

**Reply by ks-aa** [reply]
@johnshaughnessy I tried but failed... Public domain did not apply to EC2.
I gave up. Created on a public subnet.
